Formula/harmony-one.rb

```diff
@@ -0,0 +1,113 @@
+class HarmonyOne < Formula
+  desc "Harmony Blockchain Node"
+  homepage "https://github.com/harmony-one/harmony"
+  # CONTRIBUTORS: Update the following (url, sha256)
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.3.0.tar.gz"
+  sha256 "d463cd60973509f2a7e906582a09a7cd21cf3638b956f164c63e0536a2063367"
+  license "LGPL-3.0-or-later"
+
+  # CONTRIBUTORS: Versions may need to be updated according to documentation at:
+  #    -  https://github.com/harmony-one/harmony
+
+  depends_on "bash"
+  depends_on "gmp"
+  depends_on "go@1.16"
+  depends_on "jq"
+  depends_on macos: :high_sierra
+  depends_on "make"
+  depends_on "openssl@1.1"
+  
+
+  # CONTRIBUTORS: Keep resources updated according to the changelog release tag:
+  #     - https://github.com/harmony-one/harmony/releases/tag/v#{version}
+
+  resource "bls" do
+    url "https://github.com/harmony-one/bls.git",
+        revision: "2b7e49894c0f15f5c40cf74046505b7f74946e52"
+  end
+
+  resource "mcl" do
+    url "https://github.com/harmony-one/mcl.git",
+        revision: "99e9aa76e84415e753956c618cbc662b2f373df1"
+  end
+
+  def install
+    # See https://github.com/golang/go/issues/26487
+    ENV.O0 if OS.linux?
+
+    # Setup Build Paths
+    ENV["GOPATH"] = buildpath
+
+    harmony_path = buildpath/"src/github.com/harmony-one/harmony"
+    harmony_path.install buildpath.children
+
+    %w[mcl bls].each do |r|
+      temp_dir = buildpath/"src/github.com/harmony-one/#{r}"
+
+      resource(r).stage do
+        mv pwd, temp_dir
+      end
+    end
+
+    cd buildpath/"src/github.com/harmony-one/harmony"
+
+    # Correct Directory Locations In Build Scripts
+    # TODO: Switch to brew 'inreplace' formula method
+    system "sed -i -e 's/openssl/openssl\@1\.1/g' Makefile"
+    system "sed -i -e 's/openssl/openssl\@1\.1/g' scripts/go_executable_build.sh"
+
+    # Build Binarys
+    system "go", "mod", "tidy"
```

Comment:
This shouldn't be needed

Reply:
Why?
its in the readme.md for the project for building the binary
The binary doesnt build correctly without it? 
Could you please specify why it isnt needed so i can correct it

Reply:
`go mod tidy` is only needed if the dependencies are incorrectly declared. So unless the project knows it's broken and won't fix it, this isn't needed

Reply:
from the docs:
Build the harmony binary & dependent libs
go mod tidy
make
Run bash scripts/install_build_tools.sh to ensure build tools are of correct versions. If you get 'missing go.sum entry for module providing package <package_name>', run go mod tidy.